### DISCUSSION TITLE
Vim motion keys for menuing

### DISCUSSION THREAD

**Discussion by tokisuno** [monkeytypegame/monkeytype#5543] [Ideas] [answered]
I swear this used to be a feature some years ago. What happened to it? There's only one other discussion I saw where someone wanted vim motions while editing text during a test. I'm referring to when I am changing themes or languages.

**Comment by Miodec** [comment]
Looks like they got lost in a refactor somewhere. What are the keybinds again?

**Reply by tokisuno** [reply]
it was ctrl+j/k for down/up respectively while in menu.

**Reply by Miodec** [reply, marked as answer]
Ok, pushed it to the website.

**Reply by tokisuno** [reply]
works as intended. thanks! :)
